spec/lib/reporting/monthly_idv_report_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+require 'reporting/monthly_idv_report'
+
+RSpec.describe Reporting::MonthlyIdvReport do
+  let(:end_date) { Date.new(2024, 9, 1).in_time_zone('UTC').end_of_day }
+  let(:parallel) { true }
+
+  subject(:idv_report) do
+    described_class.new(end_date: end_date, parallel: parallel)
+  end
+
+  describe '#as_csv' do
+    before do
+      allow(idv_report).to receive(:reports).and_return(
+        [
+          instance_double(
+            'Reporting::IdentityVerificationReport',
+            time_range: Date.new(2024, 6, 1).all_month,
+            idv_started: 264466,
+            successfully_verified_users: 130372,
+            blanket_proofing_rates: 0.49296317863165773,
```

Comment:
why is this plural if it's a single number? maybe a good candidate to rename to singular

Reply:
There's a bunch like that:

https://github.com/18F/identity-idp/blob/main/lib/reporting/identity_verification_report.rb#L155-L169

I'll rename them while I'm in here.